src/nameclaim.h

```diff
@@ -6,15 +6,24 @@
 
 #include <vector>
 
+// This is the max claim script size, not including the script pubkey part of the script.
+// Scripts exceedeing this size are rejected in CheckTransaction in main.cpp
```

Comment:
exceedeing => exceeding

Is it obvious the units on this are bytes or should that be in the comment?